torchvision/models/regnet.py

```diff
@@ -0,0 +1,688 @@
+# Modified from
+# https://github.com/facebookresearch/ClassyVision/blob/main/classy_vision/models/anynet.py
+# https://github.com/facebookresearch/ClassyVision/blob/main/classy_vision/models/regnet.py
+
+
+import math
+import torch
+
+from collections import OrderedDict
+from functools import partial
+from typing import Any, Callable, List, Optional, Tuple
+from torch import nn, Tensor
+
+from .._internally_replaced_utils import load_state_dict_from_url
+from torchvision.models.mobilenetv2 import ConvBNActivation, _make_divisible
+from torchvision.models.efficientnet import SqueezeExcitation
+
+model_urls = {
+    # TODO(kazhang): add pretrained weights
+    "regnet_y_400m": "",
+}
+
+
+class BasicTransform(nn.Sequential):
+    """Basic transformation: [3x3 conv, BN, Relu] x2."""
+
+    def __init__(
+        self,
+        width_in: int,
+        width_out: int,
+        stride: int,
+        norm_layer: Callable[..., nn.Module],
+        activation_layer: Callable[..., nn.Module],
+    ) -> None:
+        super().__init__(OrderedDict(
+            a=nn.Sequential(
+                ConvBNActivation(width_in, width_out, kernel_size=3, stride=stride,
+                                 norm_layer=norm_layer, activation_layer=activation_layer),
+                nn.Conv2d(width_out, width_out, 3, stride=1, padding=1, bias=False),
+            ),
+            final_bn=norm_layer(width_out),
```

Comment:
Nit: This could also be replaced with `ConvBNActivation` BUT knowing that it is the last bn is beneficial as this allows us to do easily [0 gamma initialization](https://arxiv.org/abs/1812.01187). Unfortunately given that this is not supported by any of the other models, we would need to find a different generic solution to support the method. So I would be in favour of being consistent and replacing it.